web_sheet_full_width_selective/models/model.py

```diff
@@ -0,0 +1,59 @@
+# coding: utf-8
```

Comment:
Hi ! 
well, another reason is to know more easely, which models are overloaded, when you discover an OCA module. if there is ir_ui_view in ir_model.py file, it is more difficult to read.
regards.